LayoutTests/imported/w3c/web-platform-tests/media-capabilities/decodingInfo.webrtc.html

```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+<!DOCTYPE html> <!-- webkit-test-runner [ WebRTCAV1CodecEnabled=true WebRTCH265CodecEnabled=true WebRTCVP9Profile2CodecEnabled=true ] -->
```

Comment:
These changes (along with other web-platform-tests changes) will go away at the next import if not exported. I doubt this is an acceptable thing to export to the upstream repo (github.com/web-platform-tests/wpt) imo.

Reply:
Makes sense, though we have a lot of places where we do this: https://searchfox.org/wubkat/search?q=webkit-test-runner&path=LayoutTests%2Fimported%2Fw3c%2Fweb-platform-tests&case=false&regexp=false

AFAICT, the only instance that's been exported is https://github.com/web-platform-tests/wpt/blob/dfb932c07dcef9f278a21ef4ae73ca23359e3dd3/requestidlecallback/callback-timeRemaining-cross-realm-method.html (and I agree, we shouldn't be exporting our test runner modeline).

I'll ticket some cleanup here. Arguably, we should never manually enable features for WPT, because it makes tests pass in our infra that will fail in production.